src/style.css

```diff
@@ -0,0 +1,73 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+  font-style: normal;
+}
+
+.search-forms {
+  position: relative;
+  margin-top: 20px;
+}
+
+.search-text {
+  color: #3d4e61;
+  font-size: 16px;
+  font-weight: 300;
+  width: 100%;
+  height: 70px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  padding: 0 20px 0 62px;
```

Comment:
why we need this right padding? let just left `padding-left: 62px`

Reply:
done